src/components/mobileservices/BindButton.test.js

```diff
@@ -0,0 +1,55 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import BindButton from './BindButton';
+
+const setup = (propOverrides = {}) => {
+  const defaultProps = {
+    buildConfig: {
+      repoUrl: 'http://example.com',
+      branch: 'example-branch',
+      jobName: 'job-11010',
+      jenkinsfilePath: 'https://jenkins.file/path'
+    },
+    service: {
+      isBindingOperationInProgress: () => false
+    },
+    onClick: jest.fn()
+  };
+
+  const props = { ...defaultProps, ...propOverrides };
+  const wrapper = shallow(<BindButton {...props} />);
+
+  return {
+    props,
+    wrapper
+  };
+};
+
+describe('BindButton Good', () => {
```

Comment:
This should be more descriptive. Something like:

'Binding operation not in progress'